tt_metal/python_env/requirements-dev.txt

```
@@ -81,6 +81,7 @@ pydantic==2.9.2 # Required for Superset benchmarking
 fiftyone==0.25.2 # Required for Yolo benchmarking
 jiwer==3.0.5 # Required for speech recognition evaluation
 toolz==0.12.0 # Required for Save environment data step
+open_clip_torch=2.32.0 # Required for SDXL accuracy evaluation
```

```suggestion
open_clip_torch==2.32.0 # Required for SDXL accuracy evaluation
```

pls run whatever tests this is included in

---

🤦 Will try it out...